application/apps/indexer/processor/src/grabber.rs

```diff
@@ -620,3 +591,19 @@ fn subtract_what_is_possible(n: u64, m: u64) -> u64 {
         0
     }
 }
+
+/*
+    Should returns positions of all bytes, where new entity ends. It's called
+    "inlet" - point of entry's end.
+    TODO: probably it should be switched to BEGIN, but not END
+*/
+fn get_inlets_plaint_text(buffer: &[u8]) -> Result<(u64, usize), GrabError> {
+    let nl_count = bytecount::count(&buffer, b'\n') as u64;
```

Comment:
do we need this call at all? after all, it should also work without that and I guess it is not really an optimization to do this call

Reply:
Do you mean `nl_count`? It's needed to create a slot  as soon as we have to understand how many rows we have inside of bytes chunk. 
Or your are talking about function `get_inlets_plaint_text`? This function I think should become a part of `trait` to make creating of map more general.